### PR TITLE
Changes to 'Rules to Better Remote Work' 

### PR DESCRIPTION
**Pull request by joshbermanssw**
Made 4 changes as per 📋 Make a Small Content Change to SSW Rules (check-in & publish).

### CODE REVIEW

File: rules/flexible-working-hours/rule.md
Comment (jeoffreyfischer) [inline rules/flexible-working-hours/rule.md:1]: Add the rule that you have included in the text in the Related Rule.

**Review by jeoffreyfischer — COMMENTED** [review]
Just one small change

File: rules/flexible-working-hours/rule.md
Comment (joshbermanssw) [reply rules/flexible-working-hours/rule.md:1]: Done - fixed typo '/'

**Review by jeoffreyfischer — APPROVED** [review]

**Comment by jeoffreyfischer** [comment]
Make sure you give proper commit names next time

**Pull request merged**